core/src/components/datetime/datetime.scss

```diff
@@ -204,6 +204,45 @@ ion-picker-column-internal {
 // Calendar / Header / Action Buttons
 // -----------------------------------
 
+.calendar-month-year-toggle {
+  @include font-smoothing();
+  @include margin(0);
```

Comment:
A layout shift happens on MD when clicking the button

https://user-images.githubusercontent.com/2721089/216689222-f8e19d05-6d40-40a6-b51a-195dcc5b4ddc.mov

Reply:
Also, the ripple effect is missing on MD